src/gluonts/dataset/split.py

```diff
@@ -108,7 +115,10 @@ def to_integer_slice(slice_: slice, start: pd.Period) -> slice:
         return slice_
 
     if isinstance(slice_.start, pd.Period):
-        start_offset = (slice_.start - start).n
+        start_offset = ceildiv(
+            (slice_.start - start).n,
+            pd.tseries.frequencies.to_offset(start.freq).n,
+        )
```

Comment:
I think importing it like `from .. import to_offset` makes this easier to read.

Reply:
Addressed